src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -45,7 +45,7 @@
 // to distinguish one build from another. AssemblyFileVersion is specified
 // in AssemblyVersionInfo.cs so that it can be easily incremented by the
 // automated build process.
-[assembly: AssemblyVersion("1.3.1.1514")]
+[assembly: AssemblyVersion("1.3.1.1977")]
```

Comment:
Will do, I will submit a new PR based on `master` for this one too. I originally based this one off the `1.3.1` branch thinking that would be the next release (it wasn't out yet). Maybe it would be a good idea to add to the [wiki](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md) a bit about git branch management and how the team uses them, which one is current/release/etc ? I remember first PR I did was highly confusing how it was cherry-picked when merged etc.